snow/networking/handler/handler.go

```diff
@@ -435,23 +437,27 @@ func (h *handler) handleSyncMsg(ctx context.Context, msg Message) error {
 	)
 	h.resourceTracker.StartProcessing(nodeID, startTime)
 	h.ctx.Lock.Lock()
```

Comment:
Wondering if it makes sense to have a metric for how long we're waiting to acquire this lock in addition to the metrics we're adding (how long this thread holding the lock).

Reply:
As per @darioush 's comment, I think it's better to avoid adding a metric that we can derive from the other ones